.sync.yml

```diff
@@ -5,6 +5,8 @@
 .travis.yml:
   beaker_sets:
     - centos6-64
+    - centos7-64
```

Comment:
This will fail: https://github.com/theforeman/foreman-installer-modulesync/pull/121 should have the workaround.